Fails to compile sadly.
I think an `#include <string>` is missing on the top of the file. Will do a PR.

```gcc
In file included from /home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/source/Scoreboard.cpp:1:0:
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:10:8: error: ‘string’ in namespace ‘std’ does not name a type
   std::string     playerName;
        ^~~~~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:11:8: error: ‘string’ in namespace ‘std’ does not name a type
   std::string     playerClass;
        ^~~~~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:14:25: error: invalid use of ‘::’
   PlayerData(std::string);
                         ^
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:25:31: error: ‘std::string’ has not been declared
   void  loadDataFromFile(std::string);
                               ^~~~~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:26:23: error: ‘std::string’ has not been declared
   void  dumpInfo(std::string);
                       ^~~~~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:29:6: error: ‘ostream’ in namespace ‘std’ does not name a type
 std::ostream                &operator<<(std::ostream &, PlayerData const &);
      ^~~~~~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/include/Scoreboard.hpp:30:46: error: ‘string’ is not a member of ‘std’
 std::vector<PlayerData>     lineToPlayerInfo(std::string);
                                              ^~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/source/Scoreboard.cpp:3:6: error: ‘ostream’ in namespace ‘std’ does not name a type
 std::ostream    &operator<<(std::ostream &stream, PlayerData const &data)
      ^~~~~~~
/home/kellen_j/Documents/Lantio-Swords-Scrolls-and-Knuckles/source/Scoreboard.cpp:9:23: error: expected constructor, destructor, or type conversion before ‘(’ token
 PlayerData::PlayerData(std::string buf)
                       ^
CMakeFiles/ssk.dir/build.make:302: recipe for target 'CMakeFiles/ssk.dir/source/Scoreboard.cpp.o' failed
```

--- a/include/Scoreboard.hpp
+++ b/include/Scoreboard.hpp
@@ -1,6 +1,10 @@
 #ifndef SCOREBOARD_HPP_
 #define SCOREBOARD_HPP_
 
+#include <sstream>
+#include <fstream>
+#include <string>
+#include <stdexcept>
 #include <vector>
 #include <algorithm>
 

--- a/source/Scoreboard.cpp
+++ b/source/Scoreboard.cpp
@@ -21,7 +21,9 @@ PlayerData::PlayerData(std::string buf)
     playerScore = stoi(token);
   }
   catch (std::runtime_error)
+  {
     throw std::runtime_error(std::string("Reading Scoreboard file failed"));
+  }
   isScore = true;
 }
 

